samples/Common.c

```diff
@@ -12,9 +12,10 @@ VOID sigintHandler(INT32 sigNum)
     }
 }
 
-VOID onDataChannelMessage(UINT64 customData, BOOL isBinary, PBYTE pMessage, UINT32 pMessageLen)
+VOID onDataChannelMessage(UINT64 customData, PRtcDataChannel dc, BOOL isBinary, PBYTE pMessage, UINT32 pMessageLen)
```

Comment:
Though this is a sample, we still try to keep the coding standards in part with the main SDK codebase. Recommend changing dc -> pDataChannel

Reply:
Updated in dc7af43 as suggested.